pkg/promtail/targets/filetarget.go

```diff
@@ -195,18 +195,23 @@ func (t *FileTarget) run() {
 
 func (t *FileTarget) sync() error {
 
-	// Find list of directories to add to watcher.
-	path, err := filepath.Abs(t.path)
-	if err != nil {
-		return errors.Wrap(err, "filetarget.sync.filepath.Abs")
-	}
-
 	// Gets current list of files to tail.
-	matches, err := doublestar.Glob(path)
+	matches, err := doublestar.Glob(t.path)
```

Comment:
it seems that before we were passing an abs path ? but not anymore ? I was hoping @slim-bean would have more time to review this.

Reply:
@cyriltovena, absolute path is is added in for loop below: https://github.com/grafana/loki/pull/1485/files#diff-f449cd9f72b80900843b80037638cbd7R205

Previously code was assuming that only one path will be specified, now it checks if that is absolute path, and if it is not, it replaces each match with absolute path.